lib/quiet_quality/colorize.rb

```diff
@@ -0,0 +1,19 @@
+module QuietQuality
```

Comment:
Any appetite for a dependency on [another](https://github.com/fazibear/colorize) OSS project? It would add some additional text-formatting capabilities that you don't currently support as well, such as underlining or bolding.

Reply:
I had thought about it (I'm familiar with the gem), but what we needed was so minimal that I didn't bother. I don't really like that it's implemented as a String extension, mainly. If there's any problems with this implementation (different terminal types for example), I'd be willing to switch though.